src/components/Cart/Cart.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import StripeCheckout from 'react-stripe-checkout'
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+toast.configure();
+
+const Cart = (props) => {
+  const { cartItems, onAdd, onRemove, onClean, onCreateOrder } = props;
+  const totalPrice = cartItems.reduce((a, c) => a + c.id * c.cuantity, 0);
+
+  const handleToken = (token) => {
+    toast("Success! Your pokemons are on their way", { type: "success" });
+    let Order = {
+      details: token,
+      items: { cartItems, total: totalPrice }
+    }
+    onCreateOrder(Order);
+    onClean();
+  }
+
+  return(
+    <aside className="col align-self-end"> <h2> Cart items</h2>
+      <div>
+        {!cartItems.length ? <div>Cart is empty</div> : '' }
```

Comment:
Here is better to be explicit and specify `cartItems.length !== 0 ? ...` or you can do something like this:

`{cardItems.length !== 0 && <div>Cart is empty</div>}`